diff-containers/diff-containers.cabal

```diff
@@ -17,30 +17,53 @@ author:          Joris Dral
 maintainer:      operations@iohk.io, Joris Dral (joris@well-typed.com)
 category:        Data Structures
 build-type:      Simple
-tested-with:     GHC ==8.10 || ==9.2 || ==9.4 || ==9.6 || ==9.8
+tested-with:     GHC ==8.10 || ==9.6 || ==9.10
 
-library
+source-repository head
+  type:     git
+  location: https://github.com/input-output-hk/anti-diffs
+  subdir:   diff-containers
+
+flag asserts
+  description: Enable assertions
+  manual: False
+  default: False
```

Comment:
I don't think we have to add this flag. We're not using this in our code

Reply:
My point is that the flag is `False`, never set to `True`, and we don't use `-DENABLE_ASSERTIONS` in our code

If we want to run our tests in CI with assertions enabled, then IMO we should pass `-fno-ignore-asserts` in CI. Custom flags on a library are also not great

Reply:
Ah I see what you mean. True, lets remove it